cli/cmd/generate.go

```diff
@@ -54,7 +54,7 @@ var (
 	// aws command is used to generate TF code for aws
 	generateAwsTfCommand = &cobra.Command{
 		Use:   "aws",
-		Short: "Genereate terraform code for deploying into a new AWS environment",
+		Short: "Generate and optionally execute terraform code for deploying into a new AWS environment",
```

Comment:
ugh, my gut tells me that this is no longer a `Short` description 😂

Reply:
haha, i mean by comparison to the long description it's short.